test/hotspot/gtest/runtime/test_virtualMemoryTracker.cpp

```diff
@@ -1,572 +0,0 @@
-/*
```

Comment:
Why are these tests removed? Can they be adapted to the new implementation?

Reply:
They are now implemented.